README.md

```diff
@@ -128,22 +128,22 @@ app.listen(3000);
 ## Configuration
 
 The behavior of the riviere middleware can be configured by passing a configuration object,
-as argument to the `Riviere.middleware()` method call.
+as argument to the `riviere()` method call.
 
-You can start using `riviere'` by just calling `app.use(Riviere.middleware())`.
+You can start using `riviere'` by just calling `app.use(riviere())`.
 In this case the `riviere` will use its default configuration.
 
 The default configuration object is the following:
 
 ```js
-const riviereConfObj = {
+const configuration = {
     context: ctx => {
         return {};
     },
     errors: {
         enabled: true,
-        callback: (ctx, error) => {
-            throw(error);
+        callback: (err, ctx) => {
```

Comment:
To further simplify riviere in the future I would suggest removing error handling entirely. Riviere could catch errors in order to log them and rethrow them.

Reply:
👍